docs/_layouts/index.html

```diff
@@ -1,4 +1,15 @@
 ---
-layout: main
+layout: static-site-main
 ---
-<h1>{{ content }}</h1>
+<h1>{{ page.title }}</h1>
+
+{{ content }}
+
+<h2>Table of Contents</h2>
+<ul class="table-of-contents">
+  {% for child in site.posts %}
```

Comment:
What is `site.posts`? It looks like it’s going to be the list of blog posts, but I guess it should be the list of subpages?

Reply:
I'll rename this field to `subpages`. Until now, we used this field to generate blogs index and that's why it's called `posts`